src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -305,8 +305,8 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
         geo.utm2Wgs(request.destinationUTM),
         10E3
       )
-      val directMode = vehicle.mode.r5Mode.get.left.get
-      val accessMode = vehicle.mode.r5Mode.get.left.get
+      val directMode = vehicle.mode.r5Mode.flatMap(_.left.toOption).getOrElse(LegMode.valueOf(""))
```

Comment:
I just realized that these are exactly the same - why have two vals that are the same thing?

Reply:
this was just for better readability. we are passing directMode and accessMode in RoutingRequest so here we are preparing these. Now make changes in this and reduce to one variable and use parameter in RoutingRequest for identifying directMode and accessMode

Reply:
I like that much better - thanks